### PR TITLE
fix: change divider colour for caketray to match design

### PR DESCRIPTION
**Pull request by tenbradley**
### Context: 
[US122674](https://rally1.rallydev.com/#/357252966636d/custom/367300408400?detail=%2Fuserstory%2F458814721568&fdp=true?fdp=true)
The horizontal line is not the right colour currently. This updates it.

### CODE REVIEW

File: components/activity/editor/collection/d2l-activity-editor-secondary-card.js [hunk 1]
```diff
@@ -53,7 +56,7 @@ class ActivityEditorSecondaryCard extends LitElement {
 		<d2l-labs-accordion-collapse flex>
 			<span slot="header">
 				<h3>${this.titleText}</h3>
-				<hr>
+				<hr class="split">
```
Comment (zinabat) [inline components/activity/editor/collection/d2l-activity-editor-secondary-card.js:59]:
```suggestion
				<hr class="d2l-split">
```
Custom classes like this are always prefixed with `d2l-` by convention. Should probably change the other ones too.

Comment (lloydwaddell) [inline components/activity/editor/collection/d2l-activity-editor-secondary-card.js:59]: this needs to happen on the default as well

Comment (tenbradley) [reply components/activity/editor/collection/d2l-activity-editor-secondary-card.js]: It has been done.

Comment (tenbradley) [reply components/activity/editor/collection/d2l-activity-editor-secondary-card.js]: Actually, it only needs to be in the default one. The accordion the other is using has a line built in which I have switched it to use now.

**Review by zinabat — APPROVED** [review]

**Comment by tenbradley** [comment]
Closing as the card has been moved to a ui-labs repo and this is unneeded.

**Pull request closed without merging**